composer.json

```diff
@@ -41,7 +41,8 @@
     "paragonie/random_compat": "2.0.4",
     "pear/archive_tar": "1.4.14",
     "php-di/php-di": "7.0.2",
-    "ramsey/uuid": "4.7.4"
+    "ramsey/uuid": "4.7.4",
+    "shiftonelabs/laravel-sqs-fifo-queue": "^3.0"
```

Comment:
It is probably a good idea to lock this to a specific version.  We use Dependabot, so this will be updated anyway, and locking it to a specific version makes it easier to add new Composer dependencies in the future without accidentally updating others at the same time.